recipes/xsel/conda-forge.yml

```diff
@@ -0,0 +1,10 @@
+build_platform:
+  linux_aarch64: linux_64
+  linux_ppc64le: linux_64
+  osx_arm64: osx_64
+conda_build:
+  error_overlinking: true
+conda_forge_output_validation: true
+github:
+  branch_name: main
+  tooling_branch_name: main

```

Comment:
```suggestion
  tooling_branch_name: main
conda_build:
  pkg_format: '2'
conda_forge_output_validation: true
test: native_and_emulated

```

Reply:
I thought that `pkg_format: '2'` at least is the default?

Reply:
Anyways, applied.

Reply:
Everything has been synced to all recipes as adequate. @hmaarrfk

Reply:
It is with remorse that the "gamble" has no effect. I'll change the arches manually.

@hmaarrfk